<ui version="4.0" >
 <class>FunctionPlotWidget</class>
 <widget class="QWidget" name="FunctionPlotWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabParameters" >
      <attribute name="title" >
       <string>Parameters</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Function:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <property name="horizontalSpacing" >
           <number>2</number>
          </property>
          <property name="verticalSpacing" >
           <number>2</number>
          </property>
          <item row="0" column="0" >
           <widget class="QFrame" name="frame" >
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" >
             <property name="leftMargin" >
              <number>0</number>
             </property>
             <property name="topMargin" >
              <number>0</number>
             </property>
             <property name="rightMargin" >
              <number>0</number>
             </property>
             <property name="bottomMargin" >
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4" >
               <property name="text" >
                <string>constants</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbConstants" >
               <property name="toolTip" >
                <string>Choose a constant to add to the text field below</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>16</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_5" >
               <property name="text" >
                <string>functions</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbFunctions" >
               <property name="toolTip" >
                <string>Choose a function to add to the text field below</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>21</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLineEdit" name="leFunction" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QPushButton" name="bClear" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>Clear function</string>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frameAxis1" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="lAxis1" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>x-Axis:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="lAxis1Region" >
            <property name="text" >
             <string>Region</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2" >
           <widget class="QLabel" name="lAxis1Start" >
            <property name="text" >
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <widget class="QLineEdit" name="leAxis1Start" />
          </item>
          <item row="1" column="4" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="5" >
           <widget class="QLabel" name="lAxis1End" >
            <property name="text" >
             <string>End</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6" >
           <widget class="QLineEdit" name="leAxis1End" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="lAxis1Number" >
            <property name="text" >
             <string>Number of points</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3" >
           <widget class="QSpinBox" name="sbAxis1Number" >
            <property name="maximum" >
             <number>999999999</number>
            </property>
            <property name="value" >
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frameAxis2" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="lAxis2" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>y-Axis:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="lAxis2Region" >
            <property name="text" >
             <string>Region</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2" >
           <widget class="QLabel" name="lAxis2Start" >
            <property name="text" >
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <widget class="QLineEdit" name="leAxis2Start" />
          </item>
          <item row="1" column="4" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="5" >
           <widget class="QLabel" name="lAxis2End" >
            <property name="text" >
             <string>End</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6" >
           <widget class="QLineEdit" name="leAxis2End" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="lAxis2Number" >
            <property name="text" >
             <string>Number of points</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3" >
           <widget class="QSpinBox" name="sbAxis2Number" >
            <property name="maximum" >
             <number>999999999</number>
            </property>
            <property name="value" >
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTitle" >
      <attribute name="title" >
       <string>Title</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPlotStyle" >
      <attribute name="title" >
       <string>Style</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabTicks" >
      <attribute name="title" >
       <string>Tick labels</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>TODO</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
